src/apis/spawnHelpers.jsx

```diff
@@ -24,6 +24,12 @@ import { ForceDeleteModal } from "../fileActions";
 
 const options = { err: "message", superuser: "try" };
 
+const renameCommand = (o) => {
```

Comment:
Using such parameters like 'o' is bad for readability. I would much prefer that you expand the user parameters like. renameCommand = ({ selected, path, name }) => {...

Reply:
Changed :+1: